include/emp/base/error.hpp

```diff
@@ -14,9 +14,44 @@
 
 #include <iostream>
 #include <string>
+#include <sstream>
 
 namespace emp {
 
+/// TDEBUG should trigger its EMP equivalent.
```

Comment:
Is this set up in other files (i.e., emp_assert)? Don’t remember being there but might be wrong. Either way, we should be consistent between files

Reply:
Yep! That bit comes directly from `include/emp/base/_assert_trigger.hpp`: 
```
/// TDEBUG hould trigger its EMP equivalent.
#ifdef TDEBUG
#define EMP_TDEBUG 1
#endif
```